### DISCUSSION TITLE
unknown hook type status false if connecting to database (mssql/mysql/oracle etc)

### DISCUSSION THREAD

**Discussion by tanisha41** [apache/airflow#36575] [Q&A]
In airflow verion 2.5.3 I got this hook issue only for db2 but now its for all the databases. I am integrating airflow with my application but connection with any database shows this error. 

message: "Unknown hook type "mssql"", status: false}
message
: 
"Unknown hook type \"mssql\""
status
: 
false

**Comment by potiuk** [comment]
Without providing detailed stack trace, it's impossible to help you to diagnose the issue.
